src/main/java/org/sonar/plugins/csharp/CSharpSensor.java

```diff
@@ -637,20 +638,16 @@ public void consumeLine(String line) {
 
   }
 
-  private Iterable<File> filesToAnalyze() {
+  private static Iterable<File> filesToAnalyze(FileSystem fs) {
     return fs.files(fs.predicates().and(fs.predicates().hasType(Type.MAIN), fs.predicates().hasLanguage(CSharpPlugin.LANGUAGE_KEY)));
   }
 
-  private File toolInput() {
+  private static File toolInput(FileSystem fs) {
     return new File(fs.workDir(), "SonarLint.xml");
   }
 
-  private File toolOutput() {
-    return toolOutput(fs);
-  }
-
-  public static File toolOutput(FileSystem fileSystem) {
-    return new File(fileSystem.workDir(), ANALYSIS_OUTPUT_DIRECTORY_NAME);
+  private static File toolOutput(FileSystem fileSystem) {
+      return new File(fileSystem.workDir(), ANALYSIS_OUTPUT_DIRECTORY_NAME);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 5. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)